xkcd_dl/tests.py

```diff
@@ -0,0 +1,29 @@
+#! /usr/bin/python3
```

Comment:
Did you try out [py.test](http://pytest.org/latest/index.htmlhttp://pytest.org/latest/index.html) as an alternate to `unittest` module.

Will make your life much easier @BlitzKraft.
Not merging